cre/capabilities/networking/http/v1alpha/trigger.proto

```diff
@@ -20,7 +20,7 @@ message Payload {
 
 enum KeyType {
   KEY_TYPE_UNSPECIFIED = 0;
-  KEY_TYPE_ECDSA = 1;
+  KEY_TYPE_ECDSA_EVM = 1;
```

Comment:
renaming to be more specific. There are multiple implementations of ECDSA: ES256K is one instance of ECDSA and EVM has its own variation of ECDSA